src/opera/pge/disp_s1/disp_s1_pge.py

```diff
@@ -44,6 +47,43 @@ class DispS1PreProcessorMixin(PreProcessorMixin):
 
     _pre_mixin_name = "DispS1PreProcessorMixin"
 
+    def _validate_runconfig_needed_options(self):
```

Comment:
Where is this new method called from? I don't see any reference to it in `run_preprocessor`

Reply:
Whoops
Forgot to do that...